app/.gitignore

```diff
@@ -39,3 +35,7 @@ buck-out/
 \.buckd/
 android/app/libs
 android/keystores/debug.keystore
+
+# KEYS
+keys.js
+.env.js
```

Comment:
I asked you about that! You said no one looks at the .gitignore so it didn't matter.

...But I mean, no, neither are necessary. Currently not using either of those.